README.md

```diff
@@ -31,28 +31,28 @@ the form of OCI container images with [apko][apko].
 You can install Melange from Homebrew:
 
 ```shell
-brew install melange
+$ brew install melange
```

Comment:
No, we intentionally do not include the `$` prefix, because we want it to be easy for users to copy and paste commands out of the documentation.